### ISSUE TITLE
MobileNet v2

### ISSUE THREAD

**Issue by Riyya-HI** [liangfu/mxnet-mobilenet-v2#8]
I'm trying to retrain mobilenet v2 such that I already have a pre-trained model (on coco) with weights and I just want to train the last few layers with new images with objects from coco and the model just fine-tunes to that to increase accuracy without losing out on any pre-existing knowledge of all objects on coco. Basically just fine tune to new images. Not customization. How can I do this?
